app/views/api_entreprise/authorization_request_mailer/_banner_demande_valide.html.mjml

```diff
@@ -0,0 +1,27 @@
+<mj-section mj-class="section-banner" background-color="#e3ebd3">
+  <mj-column>
+    <mj-divider></mj-divider>
+    <mj-text>
+      <h1>
+        <b>
+        <% if to == "editeur" %>
```

Comment:
ce `to` est vraiment brainfuck vu qu'on a aussi un `to` passé en argument de la méthode `mail` dans le mailer, je renommerai en `entity_type` ou un truc du genre